### PR TITLE
AFKR-10: Added HIV and TB programs

### PR DESCRIPTION
**Pull request by rubailly**
- These are the default HIV and TB programs that come with bahmni

- Some concept names are short and not unique (like `HIV`) and iniz was picking the wrong concept. I chose to use UUIDs in those scenarios.

- We'll have to review how to apply translation if need

### CODE REVIEW

File: configuration/programs/programs.csv
Comment (rbuisson) [inline configuration/programs/programs.csv:3]: Let's try to be consistent:
"HIV Program Outcomes" sounds like the right label.

**Review by rbuisson — CHANGES_REQUESTED** [review]
Looks good. Working good.

Only one minor modification (see comments)
Also, as you mention, let's try to provide the translations.

File: configuration/programs/programs.csv
Comment (rubailly) [reply configuration/programs/programs.csv]: [Done](https://github.com/mekomsolutions/openmrs-config-easydoc/pull/12/commits/d2c252fbcf28bb79577fc1794490296e5831f141)

**Comment by rubailly** [comment]
After reviewing this, I found the "HIV Program Outcomes" concept doesn't exist. 
I also think the concepts used for these programs, and workflows need some review.
They've used forms/template as program names and seems confusing.

Let me have another review and see what concepts we should use and provide French translations for those concepts.

**Comment by rbuisson** [comment]
OK. I'll wait for your review. We should definitely bring all the necessary concepts and not assume anything is already present.

**Comment by rubailly** [comment]
Looks like we'll need to work on the labels as well. Some don't have the French translation
labels

**Comment by rubailly** [comment]
We'll have to enable the program display-control from the patient dashboard as well
dasboard

**Comment by rubailly** [comment]
@rbuisson The ticket was getting bigger and bigger, I left out translation for now. Let's handle it when the feature is approved and we know exactly what labels, concepts we are going to keep and then only translate those.

File: sql/program_attribute_types.sql
Comment (rbuisson) [inline sql/program_attribute_types.sql]: @rubailly could you add this file a `configuration/scripts/` path instead?

**Review by rbuisson — CHANGES_REQUESTED** [review]

**Review by rbuisson — APPROVED** [review]

File: sql/program_attribute_types.sql
Comment (rubailly) [reply sql/program_attribute_types.sql]: [Done](https://github.com/mekomsolutions/openmrs-config-easydoc/pull/12/commits/bcae562d21274d5d66d49fc8b4c3a27778037ee6?file-filters%5B%5D=)

**Pull request merged**
